travel-event-host-app/src/app/api/event/[id]/route.ts

```diff
@@ -0,0 +1,17 @@
+import { NextResponse } from 'next/server';
+import { connectMongoDB } from '@/lib/mongodb';
+import Event from '@/schemas/event';
+import mongoose from 'mongoose';
+
+export async function POST(req: Request, params: any) {
+  const id = params.id;
+  await connectMongoDB();
+  const isValidObjectId = mongoose.Types.ObjectId.isValid(id);
+  if (!isValidObjectId) {
+    return NextResponse.json({ message: 'Invalid ObjectId format' }, { status: 400 });
+  }
+
+  const eventFound = await Event.findById(id).select('-password');
```

Comment:
Can you clarify what this code does?

Reply:
that is a error, I by mistake delete that select method of the user route and added to the event route